Bottom: 50013501c4c41e834d4c507d42ca4c0f63c92884
Top:    6c5150f426cfc3836d7a99218f91adbaa7669ebf
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-10 15:21:11 +0800

Refresh of vhost-hide-used-ring-layout

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 7e4a8a7..b4378c9 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -775,12 +775,10 @@ static void handle_rx(struct vhost_net *net)
 			 * they refilled. */
 			goto out;
 		}
+		busyloop_intr = false;
 		/* On error, stop handling until the next kick. */
 		if (unlikely(err < 0))
 			goto out;
-
-		busyloop_intr = false;
-
 		if (nvq->rx_ring)
 			msg.msg_control = vhost_net_buf_consume(&nvq->rxq);
 		/* On overrun, truncate and discard */
